main/mafia.hs

```diff
@@ -197,9 +197,11 @@ commands =
  , command' "unlock" "Allow the cabal solver to choose new versions of packages again."
             (pure MafiaUnlock)
 
- , command' "quick" ( "Start the repl directly skipping cabal, this is useful "
-                   <> "developing across multiple source trees at once." )
-            (MafiaQuick <$> many pFlag <*> pGhciIncludes <*> some pGhciEntryPoint)
+ , command' "quick" ( ghciText <> "This is an alias for the \"ghci\" command." )
```

Comment:
Nit, but maybe a space before "This" given the "." is the last character in `ghciText`.